mkdocs.yml

```
@@ -19,6 +19,7 @@ plugins:
   - exclude:
       glob:
         - '**/_*'
+  - inspircd_modules: {}
```

I think this can just be:

```suggestion
  - inspircd_modules
```

---

Done.